ice/utils.py

```diff
@@ -29,6 +30,12 @@
 log = get_logger()
 
 
+def make_id() -> str:
```

Comment:
Please add a docstring to explain.

Reply:
It's an implementation detail, so I added comments